installer.sh

```diff
@@ -41,6 +41,8 @@ export IN_DOCKER=0
 # list dependencies
 export LIST_DEP=0
 export FULL=0
+# other os stuff
+export OTHER_OS=1
```

Comment:
Please default to Kali -> OTHER_OS=0
If something goes wrong it should fall back to default installation on Kali